test/ergw_http_api_SUITE.erl

```diff
@@ -270,6 +271,13 @@ http_api_delete_sessions(Config) ->
     meck_validate(Config),
     ok.
 
+http_api_delete_all_sessions() ->
+    [{doc, "Check DELETE /contexts API"}].
+http_api_delete_all_sessions(_Config) ->
+    Res1 = json_http_request(delete, "/api/v1/contexts"),
+    ?match(#{<<"contexts">> := 0}, Res1),
```

Comment:
That test is unlikely to be stable. The operation is asynchronous and the deletes are not guaranteed to be complete by the time the API call returns.